hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.js

```diff
@@ -294,3 +296,103 @@ export const PendingTable = ({ items, status }) => {
         </Container>
     );
 };
+
+export const UnconnectedBrokenTable = ({
+    items,
+    isVisible,
+    toggleVisibility,
+    onClickViewReport,
+    status,
+}) => {
+    return !items.length ? null : (
+        <Container
+            className={styles.tableContainer}
+            maxWidth={false}
+            disableGutters={true}
+        >
+            <div className={styles.title}>
+                <Typography variant="h2" className={styles.titleBroken}>
+                    Reported broken/lost items
+                </Typography>
+                <Button
+                    onClick={() => {
+                        toggleVisibility();
+                    }}
+                    color="primary"
+                >
+                    {isVisible ? "Hide all" : "Show all"}
```

Comment:
I actually don't want this ability to hide the table for the broken items table. If you broke something, I want it to be all up in your face and take up the screen. Instead of deleted all your work, you can transfer it over to `PendingTable` which currently doesn't have the button but could use it